src/components/TopBar/HamburgerMenu.jsx

```diff
@@ -37,6 +39,17 @@ const HamburgerMenu = createMenu({
         </MenuItem>,
       );
 
+      if (isExperimental) {
+        items.push(
+          <MenuItem
+            key="addOrEditInstructions"
+            onClick={isEditingInstructions ? noop : onStartEditingInstructions}
```

Comment:
When `isEditingInstructions` is true, I think we should change the Menu Item in someway so it's clear that this menu item is active.

My ideas:
1. Disable the Edit Menu Item
2. Change the text to "Stop Editing Instructions" (this will cancel and close the instructions)
3. Change the text to "Save Instructions" (this will save and close the instructions).